services/task-svc/internal/patient/patient.go

```diff
@@ -204,6 +209,16 @@ func (ServiceServer) GetRecentPatients(ctx context.Context, req *pb.GetRecentPat
 		return nil, status.Error(codes.Internal, err.Error())
 	}
 
+	// WORKAROUND: Until https://github.com/helpwave/services/issues/458 is fixed
+	if len(recentPatientIdsStrs) == 0 {
+		log.Debug().Msg("recentPatientIdsStrs was empty")
+		if patients, err := patientRepo.GetLastUpdatedPatientsForOrganization(4, organizationID); err == nil {
```

Comment:
I think we could add a separate variable for reuse

Reply:
The `4`, not reused right now. Its also just a "`WORKAROUND`".